### PR TITLE
Enhance mod tables a bit.

### PR DESCRIPTION
**Pull request by Kichura**
* Makes a change where the first words' letter of each mod description is uppercased,
* Adds parenthesis' for Fabric Capes, Iris Shaders and CEM mods instruction page references,
* Enhances descriptions for specific mods such as Entity Culling, Better Mount HUD, LambDynamicLights, etc.

**Additional infos:**

At the moment; i cannot make it so that note 1 and 2 are clickable to jump to the specific lines as GitHub only allows such jumps via blob references, meaning the readme design would be ruined in process. however the notes have been moved down to clean up tables and make them a bit prettier looking.

I also cannot confirm if these changes are deemed as an enhancement for readme file as the origin commit already has most of it properly written down but edits are welcomed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,33 +65,31 @@ The selected mods are inspired, but not limited by the [list of Optifine alterna
 | [Entity Culling](https://www.curseforge.com/minecraft/mc-mods/entityculling) | [tr9zw](https://www.curseforge.com/members/tr9zw) | &nbsp;&nbsp;✔️ | &nbsp;&nbsp;✔️ | &nbsp;&nbsp;✔️ | Stops rendering entities that are behind walls |
 | [FastChest](https://www.curseforge.com/minecraft/mc-mods/fastchest) | [fake_domi](https://www.curseforge.com/members/fake_domi) | &nbsp;&nbsp;✔️ |  |  | Makes chests render faster |
 | [FerriteCore](https://www.curseforge.com/minecraft/mc-mods/ferritecore-fabric) | [malte0811](https://www.curseforge.com/members/malte0811) | &nbsp;&nbsp;✔️ | &nbsp;&nbsp;✔️ | &nbsp;&nbsp;✔️ | Reduces RAM usage |
-| [Hydrogen](https://modrinth.com/mod/hydrogen)¹ | [CaffeineMC](https://www.curseforge.com/members/CaffeineMC) | &nbsp;&nbsp;✔️ | &nbsp;&nbsp;✔️ |  | Reduces RAM usage |
+| [Hydrogen](https://modrinth.com/mod/hydrogen)&nbsp;¹ | [CaffeineMC](https://www.curseforge.com/members/CaffeineMC) | &nbsp;&nbsp;✔️ | &nbsp;&nbsp;✔️ |  | Reduces RAM usage |
```
Comment (Madis0) [inline README.md:68]: lol why the space and especially why encoded space when it is next to the name anyway

File: README.md [hunk 2]
```diff
@@ -134,6 +130,12 @@ Mods that make other mods work. Hidden from in-game mod menu, but should not be
 | [Indium](https://www.curseforge.com/minecraft/mc-mods/indium) | [comp500](https://www.curseforge.com/members/comp500) | &nbsp;&nbsp;✔️ | &nbsp;&nbsp;✔️ | &nbsp;&nbsp;✔️ | Adds support for the Rendering API (more parity mods) |
 | [MidnightLib](https://www.curseforge.com/minecraft/mc-mods/midnightlib) | [TeamMidnightDust](https://www.curseforge.com/members/TeamMidnightDust) |  |  | &nbsp;&nbsp;✔️ | Makes some mods work |
 
+## Notes
+
+¹ [Hydrogen is not from CurseForge](https://support.overwolf.com/en/support/solutions/articles/9000197913-non-curseforge-mods),
+
+² Colormatic currently has problems, it will be re-added once resolved.
+
```
Comment (Madis0) [inline README.md:138]: not so sure about that, it can be found better when it is right below the table

File: README.md (hunk 1 above)
Comment (Kichura) [reply README.md:68]: I originally included this change to seperate it from the title as it sounded like "Hydrogen1" to me, i can undo this one if you wish.

Comment (Madis0) [reply README.md:68]: okay, feel free to keep but use a simple space for clarity

File: README.md (hunk 2 above)
Comment (Kichura) [reply README.md:138]: I seperated them from the table sections as i cannot find a neater way to put these notes elsewhere, i could go ahead and move them back but then the columns would not be quite as clean anymore.

Comment (Madis0) [reply README.md:138]: okay, show me what magic you can do with the columns first and then we'll see where to put them

**Comment by Madis0** [comment]
Thank you!

File: README.md (hunk 2 above)
Comment (Kichura) [reply README.md]: Marking this as resolved as multiple attempts were made but to no avail - this change above will have to remain sadly.

**Pull request merged**